src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/gencode/GencodeFuncotationFactory.java

```diff
@@ -2736,84 +2736,20 @@ else if ( gtfApprisTags.size() == 1 ) {
-//             case RETAINED_INTRON:			            break;
-//             case PROTEIN_CODING:			            break;
-//             case PROCESSED_TRANSCRIPT:		            break;
-//             case NON_CODING:				            break;
-//             case AMBIGUOUS_ORF:			            break;
-//             case SENSE_INTRONIC:			            break;
-//             case SENSE_OVERLAPPING:		            break;
-//             case ANTISENSE:				            break;
-             case ANTISENSE_RNA:			            return GencodeFuncotation.VariantClassification.RNA;
-             case KNOWN_NCRNA:				            return GencodeFuncotation.VariantClassification.RNA;
-//             case PSEUDOGENE:				            break;
-//             case PROCESSED_PSEUDOGENE:		            break;
-//             case POLYMORPHIC_PSEUDOGENE:	            break;
-//             case RETROTRANSPOSED:			            break;
-//             case TRANSCRIBED_PROCESSED_PSEUDOGENE:	    break;
-//             case TRANSCRIBED_UNPROCESSED_PSEUDOGENE:   break;
-//             case TRANSCRIBED_UNITARY_PSEUDOGENE:	    break;
-//             case TRANSLATED_PROCESSED_PSEUDOGENE:	    break;
-//             case TRANSLATED_UNPROCESSED_PSEUDOGENE:    break;
-//             case UNITARY_PSEUDOGENE:				    break;
-//             case UNPROCESSED_PSEUDOGENE:			    break;
-//             case ARTIFACT:					            break;
-             case LINCRNA:					            return GencodeFuncotation.VariantClassification.LINCRNA;
-             case MACRO_LNCRNA:					        return GencodeFuncotation.VariantClassification.LINCRNA;
-             case THREE_PRIME_OVERLAPPING_NCRNA:	    return GencodeFuncotation.VariantClassification.RNA;
-//             case DISRUPTED_DOMAIN:					    break;
-             case VAULTRNA:					            return GencodeFuncotation.VariantClassification.RNA;
-             case BIDIRECTIONAL_PROMOTER_LNCRNA:	    return GencodeFuncotation.VariantClassification.RNA;
-             default:
-                return GencodeFuncotation.VariantClassification.RNA;
-        }
+     * @param type A {@link String} representing a GeneTranscriptType to convert to a {@link org.broadinstitute.hellbender.tools.funcotator.dataSources.gencode.GencodeFuncotation.VariantClassification}.
+     * @return A {@link org.broadinstitute.hellbender.tools.funcotator.dataSources.gencode.GencodeFuncotation.VariantClassification} representing the given GeneTranscriptType {@link String}, or {@code null}.
+     */
+    private static GencodeFuncotation.VariantClassification convertGeneTranscriptTypeToVariantClassification (final String type ) {
+
+        //TODO: This all needs to be fixed so there is a 1:1 mapping of GencodeGtfFeature.KnownGeneBiotype->VariantClassification - Issue #4405
+        if (type.equals(GencodeGtfFeature.KnownGeneBiotype.LINCRNA.toString()) ||
+            type.equals(GencodeGtfFeature.KnownGeneBiotype.MACRO_LNCRNA.toString())) {
```

Comment:
Same comment -- if `type` can be be null reverse the order of the comparison (ie., `GencodeGtfFeature.KnownGeneBiotype.LINCRNA.toString().equals(type)`)

Reply:
👍